js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+            const quoteId = symbolParts[1];
+            const base = this.commonCurrencyCode (baseId);
+            const quote = this.commonCurrencyCode (quoteId);
+            const symbol = base + '/' + quote;
+            const active = true;
+            result.push ({
+                'id': id,
+                'symbol': symbol,
+                'base': base,
+                'quote': quote,
+                'baseId': baseId,
+                'quoteId': quoteId,
+                'active': active,
+                'info': markets[i],
+            });
+        }
+        return result;
+    }
+
+    parseTicker (ticker, market = undefined) {
+        const timestamp = this.milliseconds ();
+        const symbol = market['symbol'];
+        // ticker = ticker['ticker'];
+        const last = this.safeFloat (ticker, 'last');
+        return {
+            'symbol': symbol,
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': this.safeFloat (ticker, 'high'),
+            'low': this.safeFloat (ticker, 'low'),
+            'bid': this.safeFloat (ticker, 'buy'),
+            'bidVolume': undefined,
+            'ask': this.safeFloat (ticker, 'sell'),
+            'askVolume': undefined,
+            'vwap': undefined,
+            'open': undefined,
+            'close': last,
+            'last': last,
+            'previousClose': undefined,
+            'change': this.safeFloat (ticker, 'change'),
+            'percentage': undefined,
+            'average': undefined,
+            'baseVolume': this.safeFloat (ticker, 'volume'),
+            'quoteVolume': this.safeFloat (ticker, 'volume2'),
+            'info': ticker,
+        };
+    }
+
+    async fetchTicker (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+        };
+        const response = await this.publicGetTickers (this.extend (request, params));
+        // {
+        //     "name": "GIO/BTC",
+        //     "base_unit": "gio",
+        //     "base_fixed": 4,
+        //     "base_fee": 0.002,
+        //     "quote_unit": "btc",
+        //     "quote_fixed": 8,
+        //     "quote_fee": 0.002,
+        //     "api": true,
+        //     "base_lot": null,
+        //     "quote_lot": null,
+        //     "base_min": "0.00000010",
+        //     "quote_min": "0.00000010",
+        //     "blocks": 260722,
+        //     "block_time": "2022-01-31 14:10:08",
+        //     "wstatus": "on",
+        //     "low": "0.0000005",
+        //     "high": "0.00000058",
+        //     "last": "0.00000058",
+        //     "open": "0.000000510",
+        //     "volume": "70673.0412",
+        //     "volume2": "0.03837782973",
+        //     "sell": "0.00000058",
+        //     "buy": "0.00000055",
+        //     "at": 1643628427,
+        // },
+        if (Array.isArray (response)) {
```

Comment:
After fixing the above-mentioned issue I think you can remove this

Reply:
Ok. Resolved in latest commit [#11697](https://github.com/ccxt/ccxt/pull/11697/commits/ac050124c3ca15c30f7935008f8bc0e042e9bddd)